servant.core

```diff
@@ -147,7 +147,7 @@ targets:
     parameters : [memfile, memsize, PLL=ICE40_CORE]
     tools:
       icestorm:
-        nextpnr_options : [--hx8k, --package, tq144:4k, --freq, 16]
+        nextpnr_options : [--hx8k, --package, "tq144:4k", --freq, 16]
```

Comment:
nextpnr should be constrained to 32MHz since you changed the PLL frequency below.

Reply:
Makes sense

Reply:
I have added a commit to resolve it